conjur/cli.py

```diff
@@ -236,7 +267,7 @@ def _parse_args(parser):
 
         # Check whether we are running a command with required additional
         # arguments and if so, validate that those additional arguments are present
-        if args.resource not in ['list', 'whoami']:
+        if args.resource not in ['list', 'whoami', 'init']:
```

Comment:
`init` receives additional arguments.
The comment above suggests that if a `args.resource` is in the list of commands, then it doesn't receive additional arguments and we won't verify it

Reply:
Updated to
> Check whether we are running a command with required additional arguments/options